.gitignore

```diff
@@ -2,3 +2,4 @@ target
 Cargo.lock
 
 .cargo/config.toml
+.idea
```

Comment:
We don't add IDE-specific stuff into this file, as this is better handled by having a local, system-wide gitignore for IDE files.

Reply:
Whoops, you beat me to it.

Reply:
Fair enough, will remove this.